extensions/wikia/CommunityPage/helpers/LinkHelper.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+class LinkHelper {
+	public static function forceLoginLink( Title $title, $edit=true ) {
```

Comment:
Please add php docs for this public method. I'd suggest to remove the default value and provide it as a const instead of `bool` to make it more readable when the method is used.

What would you say for some unit tests here?